Package.swift

```diff
@@ -23,6 +23,10 @@ let package = Package(
                       checksum: "85e573e369b28b6c6829c8925a2629e62966c8f22c659863a3005f458e9bf026"),
         .target(name: "VonageClientSDKVideoMacOS",
                 path: "Sources",
+                resources: [
+                    .process("VonageClientSDKVideoMacOS/Resources/selfie_segmentation.tflite"),
```

Comment:
tflite file will no longer be available in the main SDK package after 2.28.0.
That's unless we are planning to distribute both packages (vonage-macos-sdk-2.28.0.zip and vonage-macos-sdk-transformers-2.28.0.zip) in this same SPM package.

Reply:
Are we planning on having two "names"  of Mac SPM, then why not two different pods name?
I thought the issue was final size of the app on the device. If we distribute everything n one package, it does no harm.

Reply:
That's a possibility, yes.
For pods (and the rest of the platforms) there was a team poll in which it was decided that it was more standard to have two pods (or packages for other platforms) and include or not the dependency to transformers if needed than including it by default and then having devs manually remove the framework from the dependencies if they wanted to opt out of it.
I'm not familiar with SPM so I'm not sure what would be the most common way to do it.

Reply:
@sergioturgil-vonage  
 We had a decoupling pool not a two pods one afaik anyway...

The decoupling is useful  at the "deployed" stage on the device, in reducing size if no ML is used.
As far as distributable is concerned the size is not an issue, so why not bundle everything in one package like before , instead of creating two of everything? Managing all that and communicating it will be not easy. 

@v-kpheng do we plan on having 2 pods for each complete sdk instance. Then we will have 4 pods for iOS and 4 for Mac . Too many...

We always tend to confuse decoupling with distributable size..Distributable size is not the issue here..

Reply:
Why 4 pods for iOS? I dod not understand that part